### PR TITLE
Prepare for release v1.7.6

### PR DESCRIPTION
**Pull request by joestringer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
...
+* Fix silent cilium monitor on systems with offline CPUs (Backport PR #12363, Upstream PR #12310, @pchaigno)
+* Fix syslog hook missing in DefaultLogger (Backport PR #12333, Upstream PR #12170, @ArthurChiao)
+* helm/operator: fix IPv6 liveness probe address for operator (Backport PR #12333, Upstream PR #12223, @Rolinh)
+* iptables: Remove '--nowildcard' from socket match (Backport PR #12333, Upstream PR #12248, @jrajahalme)
+* Istio integration is updated to Istio release 1.5.6. (Backport PR #12333, Upstream PR #12214, @jrajahalme)
+* Istio integration is updated to Istio release 1.5.7. (Backport PR #12357, Upstream PR #12353, @jrajahalme)
+* make: fix LOCKDEBUG env variable reference for docker-plugin-image (Backport PR #12333, Upstream PR #12318, @Rolinh)
+* option: Require native-routing-cidr only if IPv4 is enabled (Backport PR #12354, Upstream PR #12198, @brb)
+* policy/api: Add reserved:health entity (Backport PR #12333, Upstream PR #12199, @pchaigno)
+* stop Cilium from hanging on CNP or CCNP events from Kubernetes if running with 'k8s-event-handover=true' and 'kvstore=""' (Backport PR #12333, Upstream PR #12146, @aanm)
```
Comment (christarazi) [inline CHANGELOG.md]: Nit: I capitalized "stop" in the 1.6 changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+
+**CI Changes:**
+* fqdn/dnsproxy/proxy_test: increase timeout for DNS TCP client exchanges (Backport PR #12333, Upstream PR #12305, @qmonnet)
+* ginkgo-ext: Fix data-race in Writer (Backport PR #12333, Upstream PR #12025, @gandro)
+* test: Add retries to curl command (Backport PR #12357, Upstream PR #11993, @christarazi)
+* test: Download correct cilium-istioctl for the executing OS. (Backport PR #12357, Upstream PR #12109, @jrajahalme)
+* test: Skip Istio test if Ginkgo runs on unsupported runtime. (Backport PR #12357, Upstream PR #11905, @jrajahalme)
+
+**Misc Changes:**
+* [v1.7] k8s: update k8s libraries to v1.17.7 (#12208, @aanm)
```
Comment (christarazi) [inline CHANGELOG.md]: Nit: I removed "[v1.7]" from the 1.6 changelog as I thought it was redundant. Let me know if I should not do that next time.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+## v1.7.6
+
+Summary of Changes
+------------------
+
+**Minor Changes:**
+* Add possibility to configure native-routing-cidr in helm chart. (Backport PR #12354, Upstream PR #11132, @zbindenren)
+
+**Bugfixes:**
+* avoid having endpoints in 'restoring' state in case the connectivity with the KVStore is not reliable (Backport PR #12333, Upstream PR #12307, @aanm)
```
Comment (christarazi) [inline CHANGELOG.md]: Nit: if we capitalize  "stop" below, then should be consistent here as well.

**Review by christarazi — APPROVED** [review]
LGTM; nits below if we care enough

File: CHANGELOG.md (hunk 3 above)
Comment (joestringer) [reply CHANGELOG.md]: I suppose. Usually I don't bother messing with the changelogs and just encourage the original authors to consider the release notes when they write PR titles & release notes.

Will fix it up anyway.

**Pull request merged**